compiler/src/iree/compiler/Codegen/Dialect/GPU/IR/IREEGPUAttrs.cpp

```diff
@@ -1366,6 +1369,17 @@ IREE::GPU::MmaInterfaceAttr LoweringConfigAttr::getMmaKind() const {
   return getAttributes().getAs<IREE::GPU::MmaInterfaceAttr>(kMmaKindName);
 }
 
+constexpr StringLiteral kPromoteOperandsName = "promote_operands";
```

Comment:
Should we put this in a header file for the iree gpu dialect? I'd appreciate having some brief documentation explaining what these attribute mean without having to parse through the codegen logic.

Reply:
I changed it to a setter since both you and Max commented about it. I'm going to send a follow up soon to do the same to the TilingLevel strings.